lib/HTML/Form.pm

```diff
@@ -1539,7 +1539,8 @@ sub form_name_value {
     my $filename = $self->filename;
     my @headers = $self->headers;
     my $content = $self->content;
-    if (defined $content) {
+    my %headers = @headers;
+    if (defined $content || grep m/^Content$/, keys %headers) {
```

Comment:
Should this check be case insensitive?

Reply:
Good spot, thanks. I really wanted to do `keys @headers`, have never used that syntax before. Alas, we need to support old Perls.